src/utils/authentication.js

```diff
@@ -0,0 +1,46 @@
+import config from 'config';
+import passport from 'passport';
+import BearerStrategy from 'passport-http-bearer';
+import LocalStrategy from 'passport-local';
+import TokenService from '../services/token.service';
+import Sequelize from '../db/sequelize';
+import { getActiveLogger } from './winston';
+
+const authSecret = config.get('auth.secret');
+
+/**
+ * The only authorized client is Endor, who also has the secret.
+ * Endor must always authenticate users to ensure those with
+ * correct access level can access the project being requested.
+ */
+passport.use(new BearerStrategy('bearer', (accessToken, done) => {
+  if (accessToken === authSecret) {
```

Comment:
Are we essentially trying to use this like a signature?

Reply:
It is basically just a validation to check if the request came from Endor, and the only people with the secrets are Endor and Koma. 

Right @KrashLeviathan

Reply:
Correct